.github/workflows/ubuntu.yml

```diff
@@ -58,10 +58,10 @@ jobs:
           BUILDER: colcon
           ADDITIONAL_DEBS: 'libxmlrpcpp-dev'
           UNDERLAY: /root/tesseract-robotics/tesseract_ros2_target_ws/install
-          UPSTREAM_WORKSPACE: dependencies.repos
+          UPSTREAM_WORKSPACE: dependencies_${{ matrix.distro }}.repos
```

Comment:
Are your `noether` changes incompatible with `foxy`?

Reply:
Yup, I guess I can do this whole ROS version checking thing there too

Reply:
You can see the Noether foxy building failure [here](https://github.com/ros-industrial-consortium/scan_n_plan_workshop/actions/runs/6304032531/job/17114506078#step:5:189)